weaver/sdks/corda/config/dev/bootstrap-states.csv

```diff
@@ -1,118 +0,0 @@
-H, 1
```

Comment:
Is some equivalent of this file in some other location? We do initialize the state of the Corda testnets using such values, right?

Reply:
Yes in the `corda-simple-application` we do initialize the value `H,1`, but not the others. I don't know if this file was ever being used.

Reply:
Done.